app/javascript/components/workers-form/helpers.jsx

```diff
@@ -0,0 +1,237 @@
+/**
+ * Converts any value to bytes values.
+ *
+ * @param {any} value Input value.
+ * @return {number} Value in bytes.
+ */
+export const toBytes = (value) => {
+  if (!value) return undefined;
+
+  if (typeof value !== 'number') {
+    const sizeValue = value.match(/^\d+(\.\d+)*/)[0];
+    if (value.match(/gigabytes/)) {
+      return sizeValue * (1024 ** 3);
+    }
+    if (value.match(/megabytes/)) {
+      return sizeValue * (1024 ** 2);
+    }
+    if (value.match(/kilobytes/)) {
+      return sizeValue * 1024;
+    }
+    if (value.match(/.bytes/)) {
+      return Number(sizeValue);
+    }
+  }
+  return value;
+};
+
+/**
+ * Converts any value to human readable values.
+ *
+ * @param {any} value Input value.
+ * @return {string} Value in human readable format (500 MB, 2 GB, etc.).
+ */
+export const toHumanSize = (value) => {
+  if (!value) return undefined;
+
+  if (value >= 1073741824) {
+    return `${value / (1024 ** 3)} GB`;
```

Comment:
Not sure how OCD we want to be but base 2 is GiB/MiB/KiB (https://en.m.wikipedia.org/wiki/Gibibyte)

Reply:
I know, but this is how it is in the current form. Value is in GiB, etc. but label shows GB. 😖 I am not sure if we want to change it, or why it is made this way. @h-kataria

Reply:
:+1: makes sense to keep it the same as it is now

Reply:
We already had all of those helpers in `miq_formatters.js`, to match the rails formatters for charts.

We should probably merge this with those (this is `number_to_human_size`).

(`toRubyMethod` being an exception)